naiserator-dev.yaml

```diff
@@ -38,6 +38,9 @@ spec:
     - "https://syfooversiktsrv.nais.preprod.local"
   vault:
     enabled: true
+    paths:
```

Comment:
Trenger vi ikke "vanlige" secrets? `/kv/preprod/fss/syfooversiktsrv/default`. Og tilsvarende for prod.

Reply:
Det er ingen "Vanlige secrets" igjen nå etter at jeg fjernet credentials, så da la jeg ikke til det.